bootstrap/kubeadm/api/v1alpha3/conversion_test.go

```diff
@@ -60,3 +63,10 @@ func KubeadmConfigStatusFuzzer(obj *KubeadmConfigStatus, c fuzz.Continue) {
 	// KubeadmConfigStatus.BootstrapData has been removed in v1alpha4, so setting it to nil in order to avoid v1alpha3 --> v1alpha4 --> v1alpha3 round trip errors.
 	obj.BootstrapData = nil
 }
+
+func joinDNSFuzzer(obj *v1beta1.DNS, c fuzz.Continue) {
+	c.FuzzNoCustom(obj)
+
+	// DNS.Type does not exists in v1alpha4, so setting it to empty string in order to avoid v1alpha3 --> v1alpha4 --> v1alpha3 round trip errors.
+	obj.Type = "CoreDNS"
```

Comment:
According to the comment this should be
```suggestion
	obj.Type = ""
```